### PR TITLE
[Global Search] Adding clear button

### PR DESCRIPTION
**Pull request by ashleemboyer**
## 🔗 Relevant links

- [Preview link](https://dev-portal-git-ambgs-clear-button-hashicorp.vercel.app/) 🔎
- Asana tasks 🎟️
  - [[Global Search] Clear icon is missing](https://app.asana.com/0/1202097197789424/1203053381984285/f)
  - [[Global Search] Increase padding of text input to increase focus ring size](https://app.asana.com/0/1202097197789424/1203060963607153/f)

## 🗒️ What

- Adds a button for clearing command bar's text input
- Increases the padding inside of command bar's text input

## 📸 Design Screenshots

<details>
  <summary>Clear button</summary>
  <a href="https://www.figma.com/file/VD7ahvXuXWJApeGnhbW4hv/Developer?node-id=14600%3A186225&viewport=-8606%2C-17739%2C1.58">Figma link</a>
  
</details>

<details>
  <summary>Increased padding</summary>
  <a href="https://www.figma.com/file/VD7ahvXuXWJApeGnhbW4hv/Developer?node-id=14771%3A186283&viewport=-12880%2C2990%2C1.36">Figma link</a>
  
</details>

## 🧪 Testing

- [ ] Go to the home page
- [ ] Open the command bar
- [ ] It should look like:
    image
- [ ] Type a search query
- [ ] The clear button should appear like:
    image
- [ ] Hit the TAB key
- [ ] The clear button should become focused like:
    image
- [ ] Activate the clear button
- [ ] The input text should be cleared
- [ ] Focus should be returned back to the input

### CODE REVIEW

**Review by BrandonRomano — APPROVED** [review]
Had a styling suggestion but it's made with low context!

File: src/components/command-bar/components/dialog/command-bar-dialog.module.css [hunk 1]
```diff
@@ -65,6 +72,32 @@
 	}
 }
 
+.clearButtonWrapper {
+	align-items: center;
+	bottom: 0;
+	display: flex;
+	position: absolute;
+	right: 0;
+	top: 0;
```
[the remark this answers is not in the document]
Comment (ashleemboyer) [reply src/components/command-bar/components/dialog/command-bar-dialog.module.css:85]: Thanks for the suggestion! 💖

The context here is that the absolute positioning is needed to place the button on top of the input, to give the appearance that it's actually "inside" the input.

**Review by braydenlove — APPROVED** [review]
Looks good, thank you!

File: src/components/command-bar/components/dialog/command-bar-dialog.module.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (BrandonRomano) [reply src/components/command-bar/components/dialog/command-bar-dialog.module.css:85]: Ahhah makes sense, I get it now 🧠 !

**Pull request merged**